package.json

```diff
@@ -13,14 +13,17 @@
     "@react-navigation/native": "^5.9.2",
     "@react-navigation/stack": "^5.14.2",
     "expo": "~40.0.0",
+    "expo-splash-screen": "~0.8.1",
     "expo-status-bar": "~1.0.3",
+    "link": "^0.1.5",
     "react": "16.13.1",
     "react-dom": "16.13.1",
     "react-native": "https://github.com/expo/react-native/archive/sdk-40.0.1.tar.gz",
     "react-native-gesture-handler": "~1.8.0",
     "react-native-reanimated": "~1.13.0",
     "react-native-safe-area-context": "3.1.9",
     "react-native-screens": "~2.15.2",
+    "react-native-simple-splash-screen": "^1.0.1",
```

Comment:
remove this. this is of no use

Reply:
do not import any standard-less package.
see it has only 53 downloads. so we don't use such packages - https://www.npmjs.com/package/react-native-simple-splash-screen

Reply:
Ok